.github/scripts/spellcheck.words

```diff
@@ -810,6 +810,7 @@ URI
 URIs
 url
 URL's
+urlencode
```

Comment:
I disagree. This is not an English word. You would write it as URL encode.

Reply:
Ok, I'll make the change. But 'urlencoded' is used elsewhere in the document and already on the whitelist.

Reply:
Yeah, I noticed. But I think maybe those should rather be fixed as well at some point?